packages/atomic/src/components/atomic-rating/atomic-rating.tsx

```diff
@@ -1,23 +1,24 @@
 import {FunctionalComponent, h, VNode} from '@stencil/core';
 
-interface FacetValueIconRatingProps {
+interface RatingProps {
   numberOfTotalIcons: number;
   numberOfActiveIcons: number;
   icon: string;
+  iconSize?: number;
 }
 
-export const FacetValueIconRating: FunctionalComponent<FacetValueIconRatingProps> = (
-  props
-) => {
+export const Rating: FunctionalComponent<RatingProps> = (props) => {
   const width =
     ((props.numberOfActiveIcons / props.numberOfTotalIcons) * 100).toString() +
     '%';
+  const iconSize = props.iconSize ?? 0.75;
```

Comment:
I think it would be clearer to add the `rem` unit here. It would also slightly reduce the duplication on line 21.

```
const iconSize = `${props.iconSize ?? 0.75}rem`
```